.spelling

```diff
@@ -0,0 +1,111 @@
+Jetpack
+VaultPress
+WordPress
+Packagist
+WP-CLI
+rsync
+wp-admin
+config
+WordPress.com
+Jetpack-CLI
+a12s
```

Comment:
We probably should spell out Automatticians when writing in the Jetpack repo since they're public docs.